connect/src/com/telenor/connect/ui/ConnectLoginButton.java

```diff
@@ -210,4 +186,57 @@ public void onClick(View v) {
         }
 
     }
+
+    private static class MyCustomTabsServiceConnection extends CustomTabsServiceConnection {
```

Comment:
Rename
`MyCustomTabsServiceConnection` to
`WeakReferenceCustomTabsServiceConnection`?

Reply:
Sure, I have renamed them.